Add HTTP connector related docs (#1277)
wso2/docs-mi: en/docs/reference/connectors/http-connector/http-connector-example.md

# HTTP Connector Example

The HTTP Connector enables you to send HTTP or HTTPS requests within an integration sequence. 

## What you'll build

This example explains how to use the HTTP Connector to send HTTP requests effectively in your integration projects.

If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.

## Set up the integration project

> 1. Follow the steps in [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up

Reviewer: ```suggestion
1. Follow the steps in the [Create a Project]({{base_path}}/develop/create-integration-project/) documentation to set up
```

>    the Integration Project.

Reviewer: ```suggestion
   the integration project.
```

2. Create a new `HTTP` connection.

>     1. Navigate to overview page and click on **Add Artifact**.

Reviewer: ```suggestion
    1. Navigate to **Project Overview** and click on **Add Artifact**.
```

    2. Select **Connections** under **Other Artifacts**.
       
        <img src="{{base_path}}/assets/img/integrate/connectors/connections-artifact.png" title="Add connections artifact" width="800" alt="Add connections artifact"/>
   
    3. Select **HTTP** connector.
    4. Add the dependencies if not yet added.
    5. Use the following values to create the connection.
        - **Connection Name** - `JsonPlaceHolderConnection`
        - **Base URL** - `http://jsonplaceholder.typicode.com`
        - **Auth Type** - `None`

## Add integration logic

> 1. Navigate to the project overview page and click **Add Artifact**. Under **Create an Integration**, select **API**.

Reviewer: ```suggestion
1. Navigate to the **Project Overview** page and click **Add Artifact**. Under **Create an Integration**, select **API**.
```

25. In the form that opens, provide the following details:
    - **Relative Path** - `/posts/1`
    - **Headers** - ""

26. Click the **+** icon below the **DELETE** operation and add a respond mediator.

    <img src="{{base_path}}/assets/img/integrate/connectors/http-connector-example-resource-6.png" title="HTTP connector example resource 6" width="100%" alt="HTTP connector example resource 6"/>

## Get the project

You can download the ZIP file and extract the contents to get the project code.

<a href="{{base_path}}/assets/attachments/connectors/http-connector.zip">
    <img src="{{base_path}}/assets/img/integrate/connectors/download-zip.png" width="200" alt="Download ZIP">
</a>

## Deployment

Follow these steps to deploy the exported CApp in the integration runtime.

**Deploying on Micro Integrator**
To deploy and run the project, refer to the [Build and Run]({{base_path}}/develop/deploy-artifacts/#build-and-run) guide.

You can further refer the application deployed through the CLI tool. See the instructions on [managing integrations from the CLI]({{base_path}}/observe-and-manage/managing-integrations-with-micli).

> ## Test

Reviewer: For the `Test` subheading;
- Please use numbered bullets only when writing step-by-step instructions. 
- Also, let's add subheadings for each operation, similar to [this page](https://mi.docs.wso2.com/en/latest/reference/connectors/file-connector/file-connector-example/#test) for clarity.

Author: updated
